docs/source/developer_zone/dev_environment.rst

```diff
@@ -156,23 +167,39 @@ Compilation can then be launched with:
 ``libmamba`` tests
 ******************
 
-The tests for libamba are written in C++.
+The tests for libamba are written in C++ and their corresponding executables should be located at the following path once the compilation finishes:
 
 .. code:: bash
 
     ./build/libmamba/tests/test_libmamba
 
+
 ``micromamba`` integration tests
 ********************************
 
 Many ``micromamba`` integration tests are written through a Pytest Python wrapper.
 The environment variable ``TEST_MAMBA_EXE`` controls which executable is being tested.
 
+First check out the resulting executable after the copilationa like follows:
+
+.. code:: bash
+
+  $ ls build/micromamba/
+  CMakeFiles  cmake_install.cmake  mamba
+
```

Comment:
```suggestion
```

I am not in favor of adding this paragraph. The files should be there under the given configuration and the micromamba test will emit an error if `TEST_MAMBA_EXE` points to something that does not exsist.

Reply:
Maybe not, since people could have already isntalled micromamba before building, and in the previous version it is said to export `TEST_MAMBA_EXE=micromamba`, I added this just to let people to be aware of any possible name change later.

Reply:
I'm not sure to which version you refer, could you provide the link?
Otherwise, similarly to the C++ test section, we could just say somthing like: "The compiled executable get placed under `build/micromamba/micromamba" and moving on "we point the environment variable ``TEST_MAMBA_EXE`` to this location to set the executable under test" and move on to the next code section.

Reply:
At this link: https://mamba.readthedocs.io/en/latest/developer_zone/dev_environment.html, under the ` micromamba integration tests` section.
Changing to "we point the environment variable TEST_MAMBA_EXE to this location to set the executable under test"  seems to be much better for me.

Reply:
```suggestion
The compiled executable will be placed at `build/micromamba/` so we point the environment variable `TEST_MAMBA_EXE` to this location to set the executable under test

```